component-library/component-lib/src/lib/input/input.component.ts

```diff
@@ -52,6 +53,7 @@ export class InputComponent implements ControlValueAccessor, OnInit {
   @Input() formGroup = this.formGroupEmpty;
 
   disabled = false;
+  currentDisabledState: boolean = false;
```

Comment:
Is this used anywhere? If not plz remove it.

Reply:
oh right, that's not being used. will remove that. Thank you